be/src/storage/lake/tablet_reader.cpp

```diff
@@ -170,7 +170,18 @@ Status TabletReader::open(const TabletReaderParams& read_params) {
         split_morsel_queue->set_tablet_schema(_tablet_schema);
 
         while (true) {
-            auto split = split_morsel_queue->try_get().value();
+            auto split_status_or = split_morsel_queue->try_get();
```

Comment:
need to fix the following lines in scan_node.cpp as well

```
        auto maybe_morsel = morsel_queue->try_get();
        DCHECK(maybe_morsel.ok());
```

DCHECK doesn't help in RELEASE build. will crash here anyway.